bin/wls_csc.py

```diff
@@ -5,9 +5,4 @@
 from lsst.ts.ATWhiteLightSource.wlsCSC import WhiteLightSourceCSC
 from lsst.ts.ATWhiteLightSource import version
 
-WhiteLightSourceCSC.main(index=False)
-#csc = WhiteLightSourceCSC()
-#print("launching white light CSC...")
-#dtask = csc.done_task
-#asyncio.get_event_loop().run_until_complete(csc.done_task)
-#print("Done")
+WhiteLightSourceCSC.amain(index=False)
```

Comment:
I think you need to import asyncio and use `asyncio.run` here..
Also, you are not using version for anything apparently, consider removing

```
import asyncio
from lsst.ts.ATWhiteLightSource.wlsCSC import WhiteLightSourceCSC

asyncio.run(WhiteLightSourceCSC.amain(index=False))
```